Gruntfile.js

```diff
@@ -1,10 +1,11 @@
 module.exports = function( grunt ) {
-var path = require( "path" );
-require( "load-grunt-config" )( grunt, {
-	configPath: [
-		path.join( process.cwd(), "tasks/options" ),
-		path.join( process.cwd(), "tasks" )
-	],
-	init: true
-} );
+	grunt.loadNpmTasks("axe-grunt-webdriver");
```

Comment:
Hmm figured out the cause of this and opened an issue on `axe-grunt-webdriver` for a possible fix